app/src/main/java/org/intelehealth/app/activities/forgotPasswordNew/ForgotPasswordOtpVerificationActivity_New.java

```diff
@@ -54,7 +59,9 @@ public class ForgotPasswordOtpVerificationActivity_New extends AppCompatActivity
     String userUuid = "", userPhoneNum, userName;
```

Comment:
variable declaration should have proper scope defined with, private, public and protected